### PR TITLE
Fix #3181: Use finer-grained gcp workers

### LINKED ISSUE #3181: Switch to newer GCP workers
- [x] Create new set of workerType on TC Community
 - [x] Change our TC setup to use new workers

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

File: taskcluster/node-package-cpu.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ build:
   scripts:
     build: "taskcluster/node-build.sh"
     package: "taskcluster/node-package.sh"
+  workerType: "${docker.decisionTask}"
```
Comment (reuben) [inline taskcluster/node-package-cpu.yml:16]: Looks like `decisionTask` is not the best name for this worker type if it's used for caching and packaging tasks as well?

File: taskcluster/test-apk-android-25-x86_64-opt.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@ build:
     namespace: ${system.gradle_cache.namespace}
   args:
     tests_cmdline: "${system.homedir.linux}/DeepSpeech/ds/taskcluster/tc-android-apk-tests.sh x86_64 android-25 16k"
+  workerType: "${docker.dsMemTests}"
```
Comment (reuben) [inline taskcluster/test-apk-android-25-x86_64-opt.yml:21]: What does the "Mem" mean?

Comment (lissyx) [reply taskcluster/test-apk-android-25-x86_64-opt.yml:21]: Memory, as in high memory

Comment (reuben) [reply taskcluster/test-apk-android-25-x86_64-opt.yml:21]: Then please use `dsHighMemTests`.

File: taskcluster/node-package-cpu.yml (hunk 1 above)
Comment (lissyx) [reply taskcluster/node-package-cpu.yml]: changed to smallTask

**Pull request merged**